TreeMaker/python/doZinvBkg.py

```diff
@@ -63,7 +63,6 @@ def reclusterZinv(process, geninfo, residual, cleanedCandidates, suff, fastsim,
             "NumChadrons"          ,
         )
     )
-    
     process.JetsPropertiesAK8Clean.prunedMass = cms.vstring('ak8PFJetsCHSCleanPrunedMass')

```

Comment:
We are still sticking with pruned Mass for the boosted analyses, so for the control regions we still use the Pruned Mass (cleaned in the above for the photons)